### DISCUSSION TITLE
App title and minor bug.

### DISCUSSION THREAD

**Discussion by Winterwolf9729** [diasurgical/DevilutionX#5377] [General]
Hi, I have two suggestions for improvements for the app. First, I think that it would be better off simply being titled "Diablo" if that's legal because that would help to reinforce the sensation that you're playing the actual game rather than on an emulator.

Second, and more importantly, it looks to me like the game is still dealing with minor issues. All I've encountered so far is some inaccurate reporting on the number of monsters I've killed and some slight issues with control responsiveness, but I'll keep an eye out for anything else so that the app can be the best it can.

**Comment by AJenbo** [comment]
There are multiple reasons we can't name the project "Diablo". Same goes for the Android App which you appear to be specifically referencing.

> All I've encountered so far is some inaccurate reporting on the number of monsters I've killed

Can you elaborate on that? The original doesn't track kills between sessions so if that is what you are referering to it's to be expected.

> issues with control responsiveness

The game is tile based, you have to fully move to a tile before you can issue another direction, maybe this is what you are experiencing as a delay in responsiveness.

**Comment by qndel** [comment]
> 

yeah we are totally going to violate a trademark just because random dude from the internet suggested to

**Comment by AJenbo** [comment]
You could build the app yourself and change the name in Android Studio
